src/note.js

```diff
@@ -0,0 +1,20 @@
+const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+
+function sum(nbrs) {
+  let total=0;
+  if ((nbrs!== 'number')||(nbrs!== 'string')||(nbrs!== 'boolean')){return 'error'}
```

Comment:
If the data type is not correct, you should throw an exception.
You can do that by using `throw`:
`throw "Unsupported data type sir or ma'am";`